<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemForm</class>
 <widget class="QDialog" name="ItemForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Item</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #484848; font: 12px;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="5" column="1">
          <widget class="QLineEdit" name="storage_file_path_edit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #cccccc; background-color: #383838;</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_11">
           <property name="styleSheet">
            <string notr="true">color: #e2c08d;</string>
           </property>
           <property name="text">
            <string>(n) images</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">color: #e2c08d;</string>
           </property>
           <property name="text">
            <string>files.preview</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">color: #e2c08d;</string>
           </property>
           <property name="text">
            <string>* name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="12" column="1" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="set_preview_btn">
             <property name="styleSheet">
              <string notr="true">color: #81b88b;</string>
             </property>
             <property name="text">
              <string>Set Preview</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remove_image_btn">
             <property name="styleSheet">
              <string notr="true">color: #f88070;</string>
             </property>
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="8" column="1" colspan="2">
          <widget class="QLineEdit" name="tag_edit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #cccccc; background-color: #383838;</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">color: #e2c08d;</string>
           </property>
           <property name="text">
            <string>(n) storageFileSize</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="10" column="1" colspan="2">
          <widget class="QPlainTextEdit" name="comment_edit">
           <property name="styleSheet">
            <string notr="true">color: #cccccc; background-color: #383838;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">color: #e2c08d;</string>
           </property>
           <property name="text">
            <string>* category</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">color: #e2c08d;</string>
           </property>
           <property name="text">
            <string>(n) storageFilePath</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="files_usdfile_edit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #cccccc; background-color: #383838;</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_10">
           <property name="styleSheet">
            <string notr="true">color: #f88070;</string>
           </property>
           <property name="text">
            <string>(deprecated) tag</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QToolButton" name="files_preview_btn">
           <property name="styleSheet">
            <string notr="true">color: #e2c08d;</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">color: #e2c08d;</string>
           </property>
           <property name="text">
            <string>* subCategory</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="files_preview_edit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #cccccc; background-color: #383838;</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QToolButton" name="storage_file_path_btn">
           <property name="styleSheet">
            <string notr="true">color: #e2c08d;</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1" colspan="2">
          <widget class="ImageQListWidget" name="image_list">
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #cccccc; background-color: #383838;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="files_usdfile_label">
           <property name="styleSheet">
            <string notr="true">color: #e2c08d;</string>
           </property>
           <property name="text">
            <string>files.usdfile</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QToolButton" name="files_path_btn">
           <property name="styleSheet">
            <string notr="true">color: #e2c08d;</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">color: #e2c08d;</string>
           </property>
           <property name="text">
            <string>(n) status</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">color: #e2c08d;</string>
           </property>
           <property name="text">
            <string>comment</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="QLineEdit" name="storage_file_size_edit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #cccccc; background-color: #383838;</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLineEdit" name="name_edit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #cccccc; background-color: #383838;</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>name is required!</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1" colspan="2">
          <widget class="QListWidget" name="tag_list_widget">
           <property name="styleSheet">
            <string notr="true">QListWidget {
	padding: 5px;
	background-color: #383838;
}

QListWidget::item {
	border: 1px solid #518e55;
	border-radius: 3px;
	background-color: #3b5045;
	margin: 5px;
	color: white;
}

QListWidget::item::hover {
	background-color: #4d8453;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::NoDragDrop</enum>
           </property>
           <property name="movement">
            <enum>QListView::Static</enum>
           </property>
           <property name="viewMode">
            <enum>QListView::IconMode</enum>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_9">
           <property name="styleSheet">
            <string notr="true">color: #e2c08d;</string>
           </property>
           <property name="text">
            <string>(n) tags</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="1" colspan="2">
          <widget class="QComboBox" name="status_combo">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {color:#cccccc;}
QComboBox::item:selected {
	background-color: #2a2d2e;
}</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="category_combo">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {color:#cccccc;}
QComboBox::item:selected {
	background-color: #2a2d2e;
}</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QComboBox" name="sub_category_combo">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {color:#cccccc;}
QComboBox::item:selected {
	background-color: #2a2d2e;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cancel_btn">
           <property name="styleSheet">
            <string notr="true">color: #f88070;</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ok_btn">
           <property name="styleSheet">
            <string notr="true">color: #81b88b;</string>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="TagManager" name="tag_manager_widget" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageQListWidget</class>
   <extends>QListWidget</extends>
   <header>libs/imageQListWidget.h</header>
  </customwidget>
  <customwidget>
   <class>TagManager</class>
   <extends>QWidget</extends>
   <header>widgets/tagmanager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>category_combo</tabstop>
  <tabstop>sub_category_combo</tabstop>
  <tabstop>name_edit</tabstop>
  <tabstop>files_preview_edit</tabstop>
  <tabstop>files_preview_btn</tabstop>
  <tabstop>files_usdfile_edit</tabstop>
  <tabstop>files_path_btn</tabstop>
  <tabstop>storage_file_path_edit</tabstop>
  <tabstop>storage_file_path_btn</tabstop>
  <tabstop>storage_file_size_edit</tabstop>
  <tabstop>status_combo</tabstop>
  <tabstop>tag_edit</tabstop>
  <tabstop>tag_list_widget</tabstop>
  <tabstop>comment_edit</tabstop>
  <tabstop>image_list</tabstop>
  <tabstop>set_preview_btn</tabstop>
  <tabstop>remove_image_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
  <tabstop>ok_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
